### PR TITLE
Switch to using erb views

### PR DESCRIPTION
**Pull request by jcoyne**
This leverages view templates, which is where Rails typically constructs HTML.

This also gets rid of the stray `<!DOCTYPE` declarations scattered through the views.

### CODE REVIEW

File: app/controllers/embed_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class EmbedController < ApplicationController
+  append_view_path Rails.root.join('app', 'views', 'embed')
```
Comment (cbeer) [inline app/controllers/embed_controller.rb]: Is this because of the singular name?

File: app/controllers/embed_controller.rb [hunk 2]
```diff
@@ -12,7 +13,10 @@ def get
   end
 
   def iframe
-    render html: iframe_html.html_safe
+    # Trigger purl object validation (will raise Embed::PURL::ResourceNotAvailable)
```
Comment (cbeer) [inline app/controllers/embed_controller.rb]: Ew. I know this isn't you, but a `?` method raising an exception is icky.

Comment (jcoyne) [inline app/controllers/embed_controller.rb]: @cbeer Do you really want that fixed in this PR?

File: app/controllers/embed_controller.rb (hunk 1 above)
Comment (jcoyne) [inline app/controllers/embed_controller.rb]: Yes.

File: app/views/embed/body/_file.html.erb [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<div class='sul-embed-body sul-embed-file'
+      data-sul-embed-theme="<%= viewer.asset_url('file.css') %>"
```
Comment (mejackreed) [inline app/views/embed/body/_file.html.erb]: Can we drop the `viewer` prepend here? I'm seeing some strange stuff locally when assets change.

Comment (mejackreed) [inline app/views/embed/body/_file.html.erb]: Hmm.. maybe not? https://github.com/sul-dlss/sul-embed/blob/master/app/viewers/embed/viewer/common_viewer.rb#L15

Comment (jcoyne) [inline app/views/embed/body/_file.html.erb]: Sounds good for another PR. There's already a lot of moving stuff in this one. I'm trying not to change more than necessary

**Pull request merged**